backend/src/models/Product.js

```diff
@@ -1,11 +1,13 @@
 const mongoose = require("mongoose");
```

Comment:
Podes importar directamente Schema de mongoose, en vez de mongoose y luego el mongoose.Schema
```suggestion
const { Schema }= require("mongoose");
```